lib/mix/test/mix/tasks/escript_test.exs

```diff
@@ -97,6 +108,19 @@ defmodule Mix.Tasks.EscriptTest do
     end
   end
 
+  test "generate escript with debug information" do
```

Comment:
Maybe I could have the compiled escript try to get its own debug_info and print something to stdout?